lighthouse-cli/cli-flags.js

```diff
@@ -61,7 +49,14 @@ function getFlags(manualArgv) {
       .example(
           'lighthouse <url> --only-categories=performance,pwa',
           'Only run the specified categories. Available categories: accessibility, best-practices, performance, pwa, seo')
-      /**
+
+      // We only have the single string positional argument, the url.
+      .option('_', {
+        array: true,
```

Comment:
just curious why is this configured as an array?

Reply:
> just curious why is this configured as an array?

Really just for types. It's always an array, but setting the type to `string` erases the array part, this puts it back. I added a comment to make that clear.